questionnaire/utils/getOwnerOrigin/index.test.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
```

Comment:
For readability maybe have a your dummy request object defined higher in the scope, and then just copy the object (per test) and `delete` the needed properties for that particular test.

Reply:
Done. Rather than delete properties though, I've created an empty base object in a `beforeEach` and added the values I want to test.